cmd/node-termination-handler.go

```diff
@@ -24,16 +24,18 @@ import (
 	"github.com/aws/aws-node-termination-handler/pkg/config"
 	"github.com/aws/aws-node-termination-handler/pkg/drainevent"
 	"github.com/aws/aws-node-termination-handler/pkg/draineventstore"
+	"github.com/aws/aws-node-termination-handler/pkg/ec2metadata"
 	"github.com/aws/aws-node-termination-handler/pkg/node"
 	"github.com/aws/aws-node-termination-handler/pkg/webhook"
 )
 
 const (
 	scheduledMaintenance = "Scheduled Maintenance"
 	spotITN              = "Spot ITN"
+	metadataTries        = 3
```

Comment:
how do you feel about moving this outside this file? feels like its part of configuration unrelated to which monitoring functions we are tracking

Reply:
yep, that's a good idea